lib/srv/db/server.go

```diff
@@ -186,8 +187,6 @@ func (c *Config) CheckAndSetDefaults(ctx context.Context) (err error) {
 	return nil
 }
 
-// Server is a database server. It accepts database client requests coming over
```

Comment:
Nit: If you didn't think this godoc was helpful, would it make sense to make it more helpful instead of removing?

Reply:
=p Its removed by accident. Will revert